cirrus/README.md

```diff
@@ -47,6 +47,22 @@ To set up the Cirrus environment, follow these steps:
 
 By following these steps, you will create the `.env` file and configure the necessary environment variables for the Cirrus application.
 
+## Running as Non-Root User
+
+By default, the Cirrus Docker image runs the application as the root user. However, if you prefer to run the application as a non-root user for security reasons, you can build the Docker image with additional parameters.
```

Comment:
Is this true now?  Like if you don't pass in those parameters, it'll use the new 'myuser/1000/1000' user defined below, so will it ever run as root?

Reply:
I have tried it locally to build, by default the root user uses `1000` as a first user id, without any changes in other places,I mean without providing build arguments it worked, can you double check in your local to make sure? I want to make sure it doesn't impact monitor deployment.

Reply:
So on my local, on main I get

```
[app-sbAM2aaX-py3.11]jaredlockhart@Jareds-MacBook-Pro~/Documents/experimenter(main)$ docker run -ti cirrus:deploy bash
root@3a1326278208:/cirrus# id
uid=0(root) gid=0(root) groups=0(root)
```

and then in your branch i get

```
[app-sbAM2aaX-py3.11]jaredlockhart@Jareds-MacBook-Pro~/Documents/experimenter(10459/add_non_root_user)$ docker run -ti cirrus:deploy bash
cirrus@92a4a09a84dc:/cirrus$ id
uid=1000(cirrus) gid=1000(cirrus) groups=1000(cirrus)
```

so without any args it's using the new user defined in the args, which is different than the default behaviour now.  

The good news here is this is all taking place inside the container so in theory it shouldn't matter to the external consumer, and using non root inside the container is probably a good security precaution, so it should be okay to just default to always using the new non-root user inside the container, I don't think that should break existing deployments, and even if it does it's just a one time fix.

So it's probably fine to leave this as it is and then just sayin the docs 'by default it runs as cirrus/1000/1000 and if you want to override it to something else just pass these args'.

Reply:
awesome, thank you for testing that, yeah if we don't provide it uses 1000 default value,  I don't think it should break the existing deployment, but I will update the docs too, and will keep an eye on monitor logs